### DISCUSSION TITLE
Stimulsoft error while using SixLabors.Fonts

### DISCUSSION THREAD

**Discussion by tarafdarmansour** [SixLabors/Fonts#354] [Q&A] [answered]
Hi guys,
I'm trying to generate a Pdf file form a rendered stimulsoft file (mrt file) (.net 6, Stimulsoft.Reports.Web.NetCore 2023.1.1, SixLabors.Fonts 1.0.0). I'm sure that font is installed but I saw this error:
`Exception in PdfFonts.GetFontMetrics, font 'IRANSans(FaNum)' step #10: 'Method not found: 'Boolean SixLabors.Fonts.TextMeasurer.TryMeasureCharacterBounds(System.ReadOnlySpan1<Char>, SixLabors.Fonts.TextOptions, SixLabors.Fonts.GlyphBounds[] ByRef)`
Is there any HERO to help me?

**Comment by tocsoft** [comment]
we can't really support the use of the third party library but i would have to guess you have a version miss match, `Stimulsoft.Reports.Web.NetCore` is probably dependent on a beta of the `Fonts` package and the `Stimulsoft` needs updating/recompiling against our v1.0.0 release (that included breaking api call from the last beta we released)

**Reply by JimBobSquarePants** [reply]
Yeah, you've come to the wrong place. 

Report the error to Stimulsoft. You pay them for their software, and they have a duty to support it.

**Reply by tarafdarmansour** [reply]
Thank you men @tocsoft @JimBobSquarePants 
I asked my question here because they did not fully resposible for their product.
you can see an example that related to your package [here](https://forum.stimulsoft.com/viewtopic.php?t=60972) that they refered their customer to your beta package.

**Reply by JimBobSquarePants** [reply]
That's bonkers. They've never been in touch with us. 

Anyway, @tocsoft is correct. They're referencing the betas still, which is quite frankly negligent since the last update was a mere 14 days ago which is after we released v1 of both Fonts and ImageSharp.Drawing.

https://www.nuget.org/packages/Stimulsoft.Reports.Engine.NetCore/

**Reply by JimBobSquarePants** [reply]
Ouch... It's even worse than that. They're using v3 of ImageSharp but v2 compatible versions of ImageSharp.Drawing and Fonts.

<img alt="image" src="https://github.com/SixLabors/Fonts/assets/385879/7042caf4-736c-4246-ad8c-b769964397db">

**Reply by tarafdarmansour** [reply]
Thank you for your responsibility.
I hope to find a solution by changing versions.
Kind regards.

**Comment by tarafdarmansour** [comment, marked as answer]
I found the correct verson for `Stimulsoft.Reports.Engine.NetCore` version 2023.1.1:
`SixLabors.Fonts` : just `1.0.0-beta19` 
`SixLabors.ImageSharp`:  >=`2.1.3` and <= `2.1.6`
Pardon me.
I write this answer here to help others that did not find solution in Stimulsoft support community.
